chapters/annotations.tex

```diff
@@ -524,8 +540,10 @@ \section{Simulations}\label{annotations-for-simulations}\label{simulations}
 \end{annotationdefinition}
 
 \begin{annotationdefinition}[TestCase]
-\begin{synopsis}[grammar]\begin{lstlisting}
-"TestCase" "(" "shouldPass" "=" ( false | true ) ")"
+\begin{synopsis}\begin{lstlisting}
+record TestCase
+  /*literal*/ constant Boolean shouldPass;
```

Comment:
Wasn't the idea to make the defaults for all component style annotations explicit?
```suggestion
  /*literal*/ constant Boolean shouldPass = true;
```

Reply:
That sort of suggest that `TestCase.shouldPass=true` is the default, and that isn't the case as a test-case shouldn't be used in normal models (even if it has shouldPass=true).

Reply:
I have thus clarified it further.